File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/3084.java
import java.io.*;

public class Main {
    public static void main(String args[]) throws IOException{
        BufferedReader ler = new BufferedReader(new InputStreamReader(System.in));
        String linha;
        
        while((linha = ler.readLine()) != null){
            String[] ls = linha.split(" ");
            int h = Integer.parseInt(ls[0])/30, m = Integer.parseInt(ls[1])/6;
            if(h<10){
                System.out.printf("0");
            }
            System.out.printf("%d:", h);
            if(m<10){
                System.out.printf("0");
            }
            System.out.printf("%d\n", m);
        }
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/3084.cs
using System;

class URI
{
    static void Main() {
        while(true) {
            string linha = Console.ReadLine();
            if (string.IsNullOrEmpty(linha))
                break;
            string[] ls = linha.Split(' ');

            int h = int.Parse(ls[0])/30, m = int.Parse(ls[1])/6;

            if(h<10){
                Console.Write("0");
            }
            Console.Write("{0}:", h);
            if(m<10){
                Console.Write("0");
            }
            Console.Write("{0}\n", m);
        }
    }
}
